<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>941</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>941</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="loginLayout">
       <item>
        <widget class="QFrame" name="loginFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="titleLabel">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>20</y>
            <width>911</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(75, 73, 157);
font: 700 30pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>LOGIN SCREEN</string>
          </property>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>150</y>
            <width>801</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="usernameLayout">
           <item>
            <widget class="QLabel" name="usernameLabel">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(75, 73, 157);
font: 700 15pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="usernameField">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>50</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Light">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Midlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>127</red>
                   <green>127</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Mid">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>170</red>
                   <green>170</green>
                   <blue>170</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Shadow">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>220</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Light">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Midlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>227</red>
                   <green>227</green>
                   <blue>227</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>160</red>
                   <green>160</green>
                   <blue>160</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Mid">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>160</red>
                   <green>160</green>
                   <blue>160</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Shadow">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>105</red>
                   <green>105</green>
                   <blue>105</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>245</red>
                   <green>245</green>
                   <blue>245</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>220</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="WindowText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>127</red>
                   <green>127</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Light">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Midlight">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Dark">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>127</red>
                   <green>127</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Mid">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>170</red>
                   <green>170</green>
                   <blue>170</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Text">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>127</red>
                   <green>127</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="BrightText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ButtonText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>127</red>
                   <green>127</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Shadow">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>245</red>
                   <green>245</green>
                   <blue>245</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>220</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="ToolTipText">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>230</y>
            <width>801</width>
            <height>89</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="passwordLayout">
           <item>
            <widget class="QLabel" name="passwordLabel">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(75, 73, 157);
font: 700 15pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Pin number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordField">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>50</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>47</red>
                   <green>167</green>
                   <blue>218</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="loginButton">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>350</y>
            <width>201</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(168, 212, 241);
font: 700 10pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
         <widget class="QPushButton" name="back">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>71</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(168, 212, 241);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>back</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
